quirks/change-password-URLs.json

```diff
@@ -11,6 +11,7 @@
     "myaccount.ea.com": "https://myaccount.ea.com/cp-ui/security/index",
     "news.ycombinator.com": "https://news.ycombinator.com/changepw",
     "nintendo.com": "https://accounts.nintendo.com/password/edit",
+    "reddit.com": "https://www.reddit.com/settings",
```

Comment:
This is a 404 for me.

From what I can tell, the URL we want is:
https://www.reddit.com/prefs/update/